cypress/support/config/settings.js

```diff
@@ -2538,20 +2538,7 @@ module.exports = () => ({
         smoke: false,
       },
       frontPage: {
-        environments: {
-          live: {
-            paths: ['/igbo'],
-            enabled: false,
-          },
-          test: {
-            paths: ['/igbo'],
-            enabled: false,
-          },
-          local: {
-            paths: ['/igbo'],
-            enabled: false,
-          },
-        },
+        environments: undefined,
```

Comment:
We could remove the environments all together if it won't be used in the future.

Reply:
@shayneahchoon I looked into removing them but in the process of doing so I realised that other areas of this file leave the environments as undefined instead removing, so I think we have done the same for consistency 😄